test/hotspot/jtreg/serviceability/jvmti/thread/GetStackTrace/GetStackTraceAndRetransformTest/GetStackTraceAndRetransformTest.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
An Oracle copyright is not needed here if you wrote this test from scratch. If it is present then we need a comma after the copyright year please.

Reply:
Fixed